programs/svm-spoke/src/instructions/deposit.rs

```diff
@@ -209,7 +209,7 @@ pub fn deposit_v3_now(
     Ok(())
 }
 
-pub fn unsafe_deposit_v3(
+pub fn unsafe_deposit(
```

Comment:
old name `unsafe_deposit_v3` is still referenced in `programs/svm-spoke/src/state/state.rs` in comments to `pub number_of_deposits: u32`